.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # Each line is a file pattern followed by one or more owners.
 
 # global
-* @tim-mwangi @surajpuvvada @avinashkolluru @kotharironak
+* @hypertrace/trace-pipeline-owners
```

Comment:
Need to be added to repo for this to work

Reply:
I have added a write role for the above team at the repo level, but the above error is not gone. can you check the settings for this repo? Am I still missing something?

Reply:
Looks correct to me - would guess it runs it on commit and hasn't updated since your settings change. Let's see if it updates when the version is bumped.

Reply:
confirmed, valid now